spec/lib/vcr/cassette/migrator_spec.rb

```diff
@@ -1,5 +1,6 @@
 require 'tmpdir'
 require 'vcr/cassette/migrator'
+require 'yaml'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.